### PR TITLE
chore: alias of datadir.static-files and upate book cli

### PR DESCRIPTION
**Pull request by 9547**
set an alias of  `datadir.static-files`  and also update book cli

### CODE REVIEW

File: crates/node/core/src/args/datadir_args.rs [hunk 1]
```diff
@@ -20,7 +20,12 @@ pub struct DatadirArgs {
     pub datadir: MaybePlatformPath<DataDirPath>,
 
     /// The absolute path to store static files in.
-    #[arg(long = "datadir.static_files", verbatim_doc_comment, value_name = "PATH")]
+    #[arg(
+        long = "datadir.static-files",
+        alias = "datadir.static_files",
```
Comment (mattsse) [inline crates/node/core/src/args/datadir_args.rs:25]: this way we don't break anything, ty

**Review by mattsse — APPROVED** [review]
lgtm, - is the right default

**Comment by mattsse** [comment]
I think there's one more book md that we need to update

**Comment by 9547** [comment]
> I think there's one more book md that we need to update

yeah, seems the output of `make update-book-cli` made sth incorrect, try to find and fix it in another PR.

**Review by mattsse — APPROVED** [review]

**Pull request merged**